src/test/java/teammates/test/cases/ui/InstructorFeedbackDeleteActionTest.java

```diff
@@ -46,8 +46,9 @@ public void testExecuteAndPostProcess() throws Exception{
         
         assertNull(fsDb.getFeedbackSession(fs.courseId, fs.feedbackSessionName));
         assertEquals(Const.ActionURIs.INSTRUCTOR_FEEDBACKS_PAGE
-                        + "?error=false&user=idOfInstructor1OfCourse1", 
-                        r.getDestinationWithParams());
+                     + "?error=false"
+                     + "&user=idOfInstructor1OfCourse1", 
+                     r.getDestinationWithParams());
```

Comment:
This indentation may be better to show the hierarchy. i.e. lines 2-3 are parts of the first parameter while line 4 is the 2nd parameter.
```
assertEquals(Const.ActionURIs.INSTRUCTOR_FEEDBACKS_PAGE
                + "?error=false"
                + "&user=idOfInstructor1OfCourse1", 
             r.getDestinationWithParams());
```
It is also ok to combine lines 2 and 3.